src/Main/MainGui.cpp

```diff
@@ -243,13 +248,13 @@ int main(int argc, char** argv)
     }
     catch (const Base::UnknownProgramOption& e) {
         QApplication app(argc, argv);
-        QString msg = QString::fromLatin1(e.what());
-        DisplayCritical(msg);
+        QString msg = QString::fromStdString(e.what());
+        displayCritical(msg);
         exit(1);
     }
     catch (const Base::ProgramInformation& e) {
         QApplication app(argc, argv);
-        QString msg = QString::fromUtf8(e.what());
+        QString msg = QString::fromStdString(e.what());

```

Comment:
`e.what()` return `const char*`. So why are we changing this to `fromStdString`?